src/app/api/routes/users.py

```diff
@@ -26,14 +24,15 @@ async def update_my_info(payload: UserInfo, me: UserRead = Security(get_current_
 @router.put("/update-pwd", response_model=UserInfo)
 async def update_my_password(payload: Cred, me: UserRead = Security(get_current_user, scopes=["me"])):
     entry = await crud.get_entry(users, me.id)
-    await update_access_pwd(payload, entry["access_id"])
+    await crud.update_access_pwd(payload, entry["access_id"])
     return entry
 
 
 @router.post("/", response_model=UserRead, status_code=201)
+@database.transaction()
```

Comment:
I'm curious, what is the decorator doing?

Reply:
It is a way to execute queries within a transaction, see: [databases documentation](https://www.encode.io/databases/connections_and_transactions/)